GERTi/GERTiClient.lua

```diff
@@ -201,57 +292,87 @@ handler["RemoveNeighbor"] = function (eventName, receivingModem, sendingModem, p
 end
 
 handler["RegisterNode"] = function (eventName, receivingModem, sendingModem, port, distance, code, origination, tier, serialTable)
+	-- THIS IS COMPLETELY IGNORED. Until I have more knowledge about the organization, I won't touch this,
```

Comment:
Please remove this comment

Reply:
The RetrieveAddress is in fact completely ignored.

Reply:
Please remove this comment